examples/02-modal-harmonic/06-cycles_to_failure.py

```diff
@@ -0,0 +1,124 @@
+"""
+.. _ref_cycles_to_failure:
+
+Calculate the number of cycles to fatigue failure
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This example shows how to generate and use a result file to calculate the
+cycles to failure result for a simple model.
+
+Material data is manually imported, Structural Steel from Ansys Mechanical:
+
+- Youngs Modulus (youngsSteel)
+- Poisson's Ratio (prxySteel)
+- Cycles to Failure (sn_data)
```

Comment:
Should we call this `SN curve` or `Fatigue curve`? (that's the scientific name that is normally employed)

Reply:
```suggestion
- SN curve (sn_data)
```